blueprints/addon/files/.github/workflows/lint.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [main, master]
+  schedule:
+    - cron: "0 3 * * 0" # every Sunday at 3am
+
+jobs:
+  tests:
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+    name: Base Tests
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node:
+          - "12"
+          - "14"
```

Comment:
should we just chop this down to node 14.  Or even 14 and 16?

Reply:
good callout!